### ISSUE TITLE
[Feature] Enable the recent $(RATING_ICONS) variable in darkroom infoline

### ISSUE THREAD

**Issue by GLLM** [darktable-org/darktable#5019]
Hello 

Following https://github.com/darktable-org/darktable/pull/4763 which created the new (and nice) variable $(RATING_ICONS) which displays stars with UTF-8 characters, I wondered if

this variable could be made available for the darktable info line ? I'd use it as soon as it's available :)

Thank you !!

GLLM

**Comment by AlicVB** [comment]
Well... have you tried it ? because it works here and normally it was the case from the beginning...

**Comment by GLLM** [comment]
I did ... Not used to ask or report something without trying (it's the kind of things which usually drives me mad when I see it, at work).

Did I try it with a faulty set of pictures ... ? I don't know.
Between my FR & this message, I think I updated dt 3 or 4 times times. Was something broken at some point ... I dont know.
Seeing your message, I've tried it again and it worked.

Sorry for this crappy F.R. @AlicVB
My apologies.

Closing it.
